### PR TITLE
feat: update readme with OSS bits [ROAD-330]

### PR DESCRIPTION
**Pull request by michelkaporin**

### CODE REVIEW

**Review by bastiandoetsch — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,124 +1,147 @@
...
+
+You can filter the issues by setting the severities you want to see using the `snyk.severity` setting. E.g. set `"snyk.severity": { "critical": true, "high": true, "medium": true, "low": false }` to hide low severity issues. You can also apply the setting via Settings UI.
 
 ### Editor window
 
 The editor window (in the middle of the results screen) shows the code that is inspected. This ensures that when you are
 inspecting a Snyk issue, you always have the code context close to the issue.
 
-### Snyk suggestions window
+### Snyk Code vulnerability window
```
Comment (deepcodeg) [inline README.md:144]: Yep, a screenshot will help here as well.

File: README.md [hunk 2]
```diff
@@ -130,16 +153,31 @@ example variable names of your code and the line numbers in red. You can also se
 - You can add ignore comments that would make Snyk ignore this particular suggestion, or all of these suggestions for
   the whole file, by using the two buttons on the lower end of the panel.
 
-We also include a feedback mechanism to report false positives so you others do not see the same issue.
+We also include a feedback mechanism to report false positives so that others do not see the same issue.
+
+### Snyk OSS vulnerability window
 
+OSS vulnerability tab shows information about the vulnerable module.
```
Comment (deepcodeg) [inline README.md:160]: One would be greatly appreciated here as well.

File: README.md (hunk 1 above)
Comment (michelkaporin) [reply README.md:145]: ✅

**Pull request merged**